genologics/lims.py

```diff
@@ -564,7 +564,7 @@ def get_batch(self, instances, force=False):
             for node in root.getchildren():
                 instance = instance_map[node.attrib['limsid']]
                 instance.root = node
-        return instance_map.values()
+        return list(instance_map.values())
```

Comment:
Here we *should* keep the `list` to maintain the contract to the caller.

Compared to https://github.com/Molmed/genologics/pull/17/files#r610544226 where we're just iterating, that doesn't require the `list` call.